src/main/java/org/mifos/pheevouchermanagementsystem/data/VoucherValidator.java

```diff
@@ -128,12 +128,12 @@ public PhErrorDTO validateVoucherLifecycle(RequestDTO request) {
             request.setVoucherInstructions(new ArrayList<>(Arrays.asList(new VoucherInstruction())));
         }
 
-//        request.getVoucherInstructions().forEach(voucherInstruction -> {
-//            // Check for serialNumber
-//            validatorBuilder.reset().resource(resource).parameter(serialNumber).value(voucherInstruction.getSerialNumber())
-//                    .isNullWithFailureCode(VoucherValidatorsEnum.INVALID_SERIAL_NUMBER).validateFieldMaxLengthWithFailureCodeAndErrorParams(
-//                            expectedSerialNumberLength, VoucherValidatorsEnum.INVALID_SERIAL_NUMBER_LENGTH);
-//        });
+        // request.getVoucherInstructions().forEach(voucherInstruction -> {
```

Comment:
delete these lines if not needed

Reply:
these are some validation code which might be use later